.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+**Thank you for submitting this pull request**
+
+**JIRA**: _(please edit the JIRA link if it exists)_ 
+
+[link](https://examle.com)
+
+**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_
+
+* referenced pull request 1
+* referenced pull request 2
+* referenced pull request 2
+etc. 
+
+<pre>
+How to retest a PR or trigger a specific build:
+
+* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
+ 
+* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
+  
+* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>
+
+* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>
+
+* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
+</pre>
```

Comment:
Is the section how to test PR valid here?

Reply:
yes: could be Jenkins retest this or Jenkins test this - since "test this" is hard coded in the ghprb plugin

Reply:
Did some fixes. It works. FDB, CDB and others don't make sense.